db/bigtable_eth1.go

```diff
@@ -2113,11 +2174,30 @@ func (bigtable *Bigtable) GetAddressTransactionsTableData(address []byte, search
 		pageToken = fmt.Sprintf("%s:I:TX:%x:%s:", bigtable.chainId, address, FILTER_TIME)
 	}
 
-	transactions, lastKey, err := BigtableClient.GetEth1TxForAddress(pageToken, 25)
+	transactions, keys, err := BigtableClient.GetEth1TxForAddress(pageToken, 25)
 	if err != nil {
 		return nil, err
 	}
 
+	idxs := make([]int64, len(keys))
+	for i, k := range keys {
+		tx_idx, err := strconv.Atoi(strings.Split(k, ":")[6])
+		if err != nil {
+			return nil, fmt.Errorf("error parsing Eth1InternalTransactionIndexed tx index: %v", err)
+		}
+		tx_idx = 10000 - tx_idx
```

Comment:
**Question:** This hardcoded 10000 should be a constant for better readability. Also applies to https://github.com/gobitfly/eth2-beaconchain-explorer/blob/133e51a61a11bab1dd3fce36676e2a70befe7ff1/db/bigtable_eth1.go#L2647.

If I am not mistaken, this comes from places like here: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/133e51a61a11bab1dd3fce36676e2a70befe7ff1/db/bigtable_eth1.go#L1049
Those then of course should also use the new constant.

WDYT?

Reply:
Yes, totally! I'm also a big proponent of avoiding all "magic numbers" everywhere. Didn't do it because, well, if that's how it's done I thought  I might shouldn't change it - but I'll gladly do.